radicle-httpd/src/api/test.rs

```diff
@@ -0,0 +1,133 @@
+use std::path::Path;
+use std::sync::Arc;
+use std::{env, fs};
+
+use axum::body::Body;
+use axum::http::Request;
+use axum::response::Response;
+use axum::Router;
+use rand::distributions::{Alphanumeric, DistString};
+use serde_json::Value;
+use tower::ServiceExt;
+
+use radicle::cob::issue::Issues;
+use radicle::git::raw as git2;
+use radicle::storage::WriteStorage;
+use radicle_cli::commands::rad_init;
+
+use crate::api::Context;
+
+pub const HEAD: &str = "1e978d19f251cd9821d9d9a76d1bd436bf0690d5";
+pub const HEAD_1: &str = "f604ce9fd5b7cc77b7609beda45ea8760bee78f7";
+
+pub fn seed() -> Context {
+    let random_suffix = Alphanumeric.sample_string(&mut rand::thread_rng(), 16);
+    let temp_dir = env::temp_dir()
+        .join("radicle-httpd-tests")
+        .join(random_suffix);
+    let workdir = temp_dir.join("hello-world");
+    let rad_home = temp_dir.join("radicle");
+
+    env::set_var("RAD_PASSPHRASE", "asdf");
+    env::set_var("RAD_DEBUG", "1");
```

Comment:
Are these needed? We try to avoid env vars in tests if possible, for obvious reasons.

Reply:
Yes, they are needed, I think `rad init` needs them.

Reply:
That's strange, can you double check?

Reply:
Yea, I checked, if I remove them, prompt asks for password in tests, and if I remove debug, the identity won't be stable.

Reply:
What prompts you?

Reply:
I think I'm going to look into a more sustainable way of doing this, for now, can you set it to "radicle"? Same passphrase we use in the CLI tests. Because otherwise the test will fail if run in parallel with the CLI test.

Reply:
[fixed](https://github.com/radicle-dev/heartwood/compare/3ab407822c6d0f3bc12a7b001ae24cf716eb8600..9c3fff3f60b89ece2b1281fbcb6be961adf8faf8)